src/pages/CategorizationPage.js

```diff
@@ -32,7 +36,7 @@ CategorizationPage.propTypes = {
         parties: PropTypes.arrayOf(PropTypes.string)
       })
     )
-  }).isRequired,
+  }),
```

Comment:
I will change it back to `isRequired`. The reason I changed it was because there is the conditinal rendering
```
{!userState ? (
  <Typography variant="h4">Select a file and bank on import page</Typography>
) : (
  <Categorizer userState={userState} updateCategories={updateCategories} />
)}
``` 
and therefore it seemed like we are allowing undefined userState. However, now when I think about it more, there never is a situation where userState would be undefined. 

We should at some point probably also make it so that the user can NOT navigate to all pages, if some data is missing. Then we would not have to deal with these